### PR TITLE
RPRBLND-2004: add rprDiskLightSetInnerAngle (core 2.2.9)

### PR DESCRIPTION
**Pull request by DagerD**
### PURPOSE
Add support for Spot Blend for core 2.2.9.

### EFFECT OF CHANGE
Added support for "Spot Blend" Spot Light property.

### TECHNICAL STEPS
Implemented DiskLightSetInnerAngle for 'FULL2' render quality

### CODE REVIEW

File: src/rprblender/export/light.py [hunk 1]
```diff
@@ -145,6 +145,9 @@ def sync(rpr_context: RPRContext, obj: bpy.types.Object, instance_key=None):
         oangle = 0.5 * light.spot_size  # half of spot_size
         iangle = oangle * (1.0 - light.spot_blend * light.spot_blend)  # square dependency of spot_blend
         rpr_light.set_cone_shape(iangle, oangle)
+        
+        if rpr_context.blender_data['depsgraph'].scene.rpr.render_quality == 'FULL2':
+            rpr_light.set_inner_angle(iangle)
```
Comment (bsavery) [inline src/rprblender/export/light.py:150]: Hybrid pro has no such method.  Won't this cause an error?

Comment (bsavery) [reply src/rprblender/export/light.py:150]: Nevermind.  I see the if statement.

File: src/rprblender/export/light.py [hunk 2]
```diff
@@ -145,6 +145,9 @@ def sync(rpr_context: RPRContext, obj: bpy.types.Object, instance_key=None):
         oangle = 0.5 * light.spot_size  # half of spot_size
         iangle = oangle * (1.0 - light.spot_blend * light.spot_blend)  # square dependency of spot_blend
         rpr_light.set_cone_shape(iangle, oangle)
+        
+        if rpr_context.blender_data['depsgraph'].scene.rpr.render_quality == 'FULL2':
```
Comment (bnagirniak) [inline src/rprblender/export/light.py:149]: try: `if isinstance(rpr_context, RPRContext2):`

**Review by bnagirniak — COMMENTED** [review]
code improvement

**Review by bnagirniak — APPROVED** [review]
Looks good.

**Comment by ARozghon** [comment]
working correctly with Full and Interactive mode

**Pull request merged**